lib/src/main/kotlin/se/gustavkarlsson/koptional/Optional.kt

```diff
@@ -4,6 +4,7 @@ sealed class Optional<out T : Any> {
 
     abstract val value: T?
 
+    @Deprecated(message = "Unsafe to use. Use `ifPresent` or `ifAbsent` instead")
```

Comment:
`ifPresent` and `ifAbsent` are not really alternatives to `valueUnsafe` . They are rather alternatives to use cases that might use `valueUnsafe`.

I would change it to just "Useless" and add `ReplaceWith("value!!")` or something like that.